pkg/network/namescheme/networknamescheme.go

```diff
@@ -138,6 +138,16 @@ func PodHasOrdinalInterfaceName(podNetworkStatus map[string]networkv1.NetworkSta
 	return false
 }
 
+// PodHasOrdinalInterfaceName2 checks if the given pod network status has at least one pod interface with ordinal name
```

Comment:
What is the end goal of `PodHasOrdinalInterfaceName2`? Is it supposed to supplant the original `PodHasOrdinalInterfaceName2`?

If it is a replacement and you're aiming to supplant the latter, consider adding a deprecation notice so that people prefer using `PodHasOrdinalInterfaceName2`. Personally I think the additional effort of establishing a distinction between the two is worth it (e.g., `PodHasOrdinalIfaceNameInMap`, `PodHasOrdinalIfaceNameInSlice`)

Reply:
> What is the end goal of `PodHasOrdinalInterfaceName2`? Is it supposed to supplant the original `PodHasOrdinalInterfaceName`?

A follow-up PR to replace the original `PodHasOrdinalInterfaceName` with this one is in the works.

> If it is a replacement and you're aiming to supplant the latter, consider adding a deprecation notice so that people prefer using `PodHasOrdinalInterfaceName2`. Personally I think the additional effort of establishing a distinction between the two is worth it (e.g., `PodHasOrdinalIfaceNameInMap`, `PodHasOrdinalIfaceNameInSlice`) regardless.

Since the number contribution to sig-network related code is small, and the follow-up is in the works, I don't think its worth the effort to mark the original function as deprecated.